db/dataset_eval.py

```diff
@@ -11,12 +11,14 @@
 STATUS_RUNNING = "running"
 STATUS_DONE = "done"
 STATUS_FAILED = "failed"
-
+# Eval locations are defined in `eval_location_type` type. See schema definition.
```

Comment:
I did that because there were similar comments for the other constants. However the one you suggested looks better.